traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go

```diff
@@ -63,14 +66,26 @@ func BuildWhereAndOrderBy(parameters map[string]string, queryParamsToSQLCols map
 			log.Debugln("Incorrect name for orderby: ", orderby)
 		}
 	}
+	// needs error checking
+	if limit, exists := parameters["limit"]; exists {
+		log.Debugln("limit: ", limit)
+		paginationClause += " " + limit
+		if offset, exists := parameters["offset"]; exists {
+			paginationClause += BaseOffset + " " + offset
```

Comment:
This type of query building is unsafe. In testing I was able to run arbitrary code in the `limit` clause by submitting a request like e.g.: `toget '1.4/servers?limit=1%3B%20DROP%20TABLE%20server%20CASCADE%3B%20--&orderBy=host_name'` which is something that even a read-only user can do.